tests/runtests.py

```diff
@@ -1,10 +1,22 @@
+#!/usr/bin/env python
 import os
 import sys
 
 sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "test_sqlite")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings.sqlite")
+
+SOCK_SETTINGS = "settings.sqlite_usock"
 
 if __name__ == "__main__":
+    if any(SOCK_SETTINGS in i for i in sys.argv) or SOCK_SETTINGS in os.environ.get(
```

Comment:
Any ideas on how to enable tests with unix socket?

Reply:
They work locally now, we probably need to move the CI to use the start redis script which can pass the arguments needed to configure redis. Alternatively we could probably use socat to pipe the tests to redis...